src/main/java/net/glowstone/GlowWorld.java

```diff
@@ -1820,4 +1820,21 @@ else if (tickMap.containsKey(target))
     public void cancelPulse(GlowBlock block) {
         requestPulse(block, 0);
     }
+
+    @Override
+    public int hashCode() {
+        return getUID().hashCode();
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == null) {
+            return false;
+        }
+        if (getClass() != obj.getClass()) {
+            return false;
+        }
+        GlowWorld other = (GlowWorld) obj;
+        return getUID() == other.getUID();
```

Comment:
I'm not sure if it is possible that one world can have multiple instances. @mastercoms would need to check this?